docs/devices/9290024693.md

```diff
@@ -29,7 +29,17 @@ pageClass: device-page
 
 ## Pairing
 New Bulbs are automatically in pairing mode. 
-Factory resetting a Hue bulb can be accomplished in 5 ways which are described below. After resetting the bulb will automatically connect.
+Factory resetting a Hue bulb can be accomplished in 6 ways which are described below. After resetting the bulb will automatically connect.
+
+### Power Cycling
```

Comment:
Please move above `<!-- Notes END: Do not edit below this line -->`

Reply:
I'm going to open a new PR, I didn't realise this was auto-generated, so I'm going to edit the proper source